packages/wonder-blocks-spacing/docs.md

```diff
@@ -1,42 +1,93 @@
+const {Strut} = require("./index.js");
 
-<div>
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+        alignItems: "center",
+        marginBottom: Spacing.xxSmall,
+    },
+});
+
+<View>
     {Object.keys(Spacing).map((spaceName, idx) => (
-        <div
+        <View
             key={idx}
-            style={{
-                display: "flex",
-                alignItems: "center",
-                marginBottom: Spacing.xxSmall,
-            }}
+            style={styles.row}
         >
-            <div style={{
+            <View style={{
                 width: 250,
-                paddingRight: Spacing.xSmall,
                 textAlign: "right",
             }}>
                 {spaceName}: {Spacing[spaceName]}px
-            </div>
-            <div style={{width: Spacing.xxxLarge, marginRight: Spacing.xSmall}}>
-                <div
+            </View>
+            <Strut size={Spacing.xSmall}/>
+            <View style={{width: Spacing.xxxLarge}}>
+                <View
                     style={{
                         backgroundColor: "black",
                         width: Spacing[spaceName],
                         height: Spacing.xxxSmall,
                     }}
                 />
-            </div>
-            <div
+            </View>
+            <Strut size={Spacing.xSmall}/>
+            <View
                 style={{
                     backgroundColor: "black",
                     width: Spacing.xxxSmall,
                     height: Spacing[spaceName],
                 }}
             />
-        </div>
+        </View>
     ))}
-</div>
+</View>
+```
+
+## Spring & Strut
+
+`Spring` and `Strut` are two components that can make certain layouts easier to implement.
+`Spring` is infinitely compressible and expands to fill available space while `Strut`
+is uncompressible and occupies a fixed amount of space specified by its `size` prop.
+
+```js
+const {StyleSheet} = require("aphrodite");
+const {View, addStyle} = require("wonder-blocks-core");
+const Color = require("wonder-blocks-color");
+const Spacing = require("./index.js").default;
+const {Spring, Strut} = require("./index.js");
+
+// TODO(kevinb): replace with wonder-blocks-button once yarn workspaces land
+const Button = addStyle("button");
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: "row",
+        border: `solid 1px ${Color.offBlack50}`,
+    },
+    button: {
+        fontSize: 24,
+        width: 100,
+        borderRadius: 4,
+        border: "none",
+        height: 40,
+    },
+});
+
+<View style={styles.container}>
+    <Button style={styles.button}>A</Button>
+    <Strut size={Spacing.small}/>
+    <Button style={styles.button}>B</Button>
+    <Strut size={Spacing.small}/>
+    <Button style={styles.button}>C</Button>
+    <Spring/>
+    <Button style={styles.button}>Cancel</Button>
+    <Strut size={Spacing.small}/>
+    <Button style={styles.button}>Accept</Button>
+</View>
```

Comment:
This will look cleaner once we're able to import `wonder-blocks-button` as a dev dependency.